### ISSUE TITLE
Can I use your atlas.png file?

### ISSUE THREAD

**Issue by Romainrob** [ptitSeb/stuntcarremake#10]
Hi! Bonjour!
Sorry to open a ticket, but I'm not sure how else to contact you.
I'd like to ask the permission to use your atlas.png file.
I'm building an "animated" SimHub Overlay as a fun tribute to this great game.
I don't plan to sell it but I'd like to freely share it to the SimHub community.
I'll mention your work of course.
Please. Let me know if it's ok with you.
Have a nice day!

**Comment by ptitSeb** [comment]
Yes sure, no problem. Give me the link when it's done so I can see the
result 😊

Le lun. 16 sept. 2019 à 17:47, Romainrob <notifications@github.com> a
écrit :

**Comment by Romainrob** [comment]
Sweet! I will.
Don't expect much, it's a fun project.
Thank you so much!

**Comment by Romainrob** [comment]
Here is the overlay :https://www.youtube.com/watch?v=ZExdKNoKNw4
I couldn't link your remake on RaceDepartment (it's against the TOS) so I did it in the video description. It's good fun and a lot of memories :).
Thanks again!

**Comment by ptitSeb** [comment]
Look nice :D !
Thanks for this.

**Comment by Romainrob** [comment]
I'm glad you like it.
Here it is on a fitting track:
https://www.youtube.com/watch?v=y7qdgyWWN-w&feature=youtu.be
